app/scripts/search/search.controllers.ts

```diff
@@ -225,6 +226,11 @@ module ngApp.search.controllers {
       this.$state.go("query." + toTab, stateParams, { inherit: true });
     }
 
+    toggleFacets(shouldCollapse) {
```

Comment:
Sorry missed this.. it was only needed to send a message to the pie chart directive to refresh itself, and projects/annotations don't have pie charts.